basic_autonomy/src/basic_autonomy.cpp

```diff
@@ -75,9 +75,34 @@ namespace basic_autonomy
             std::vector<PointSpeedPair> points_and_target_speeds;
 
             cav_msgs::LaneFollowingManeuver lane_following_maneuver = maneuver.lane_following_maneuver;
-            
+
             auto lanelets = wm->getLaneletsBetween(starting_downtrack, lane_following_maneuver.end_dist + detailed_config.buffer_ending_downtrack, true, true);
 
+
+            bool lanelets_defined = !maneuver.lane_following_maneuver.lane_ids.empty();
```

Comment:
These changes need to be applied to basic_autonomy_ros2 as well.

Reply:
We can wait on this, since we'll be address the other basic_autonomy changes again next sprint.